lock-api/src/main/java/com/palantir/lock/LockServerOptions.java

```diff
@@ -48,6 +48,7 @@ public class LockServerOptions implements Serializable {
      * client accessing it.
      */
     @Value.Default
+    @JsonProperty("isStandaloneServer")
     public boolean isStandaloneServer() {
```

Comment:
we should probably have serDe tests for these types

Reply:
do you have an idea about how we'd do that? the problem here is that infra mappers (conjure/wc) used for serDe aren't obvious in the forefront 

I think we have integration tests where this would have been caught, but I think we might not have coverage for this endpoint (it's not used a lot is my understanding)

Reply:
I was thinking something like the following as a regression test (e.g. to catch if immutables style changed):

```
public class LockServerOptionsTest {

    private final ObjectMapper serverMapper = ObjectMappers.newServerObjectMapper();
    private final ObjectMapper clientMapper = ObjectMappers.newClientObjectMapper();

    @Test
    public void jsonSerDe() throws Exception {
        LockServerOptions lockServerOptions =
                LockServerOptions.builder().standaloneServer(false).build();

        String json = serverMapper.writeValueAsString(lockServerOptions);
        assertThat(json).isNotEmpty().contains("\"isStandaloneServer\":false");

        assertThat(clientMapper.readValue(json, LockServerOptions.class)).isEqualTo(lockServerOptions);
        assertThat(serverMapper.readValue(json, LockServerOptions.class)).isEqualTo(lockServerOptions);
    }

    @Test
    public void defaultJsonSerDe() throws Exception {
        LockServerOptions lockServerOptions = LockServerOptions.DEFAULT;
        String json = serverMapper.writeValueAsString(lockServerOptions);
        assertThat(json).isNotEmpty().contains("\"isStandaloneServer\":true");

        assertThat(clientMapper.readValue(json, LockServerOptions.class)).isEqualTo(lockServerOptions);
        assertThat(serverMapper.readValue(json, LockServerOptions.class)).isEqualTo(lockServerOptions);
    }

    @Test
    public void deserializeExistingOptions() throws Exception {
        String json = "{" + //
                "\"maxAllowedLockTimeout\":{\"time\":1,\"unit\":\"MILLISECONDS\"},"
                + "\"maxAllowedClockDrift\":{\"time\":2,\"unit\":\"SECONDS\"},"
                + "\"maxAllowedBlockingDuration\":{\"time\":3,\"unit\":\"MINUTES\"},"
                + "\"maxNormalLockAge\":{\"time\":4,\"unit\":\"HOURS\"},"
                + "\"randomBitCount\":32,"
                + "\"stuckTransactionTimeout\":{\"time\":5,\"unit\":\"DAYS\"},"
                + "\"slowLogTriggerMillis\":12345,"
                + "\"isStandaloneServer\":false"
                + "}";

        LockServerOptions lockServerOptions = LockServerOptions.builder()
                .maxAllowedLockTimeout(SimpleTimeDuration.of(1, TimeUnit.MILLISECONDS))
                .maxAllowedClockDrift(SimpleTimeDuration.of(2, TimeUnit.SECONDS))
                .maxAllowedBlockingDuration(SimpleTimeDuration.of(3, TimeUnit.MINUTES))
                .maxNormalLockAge(SimpleTimeDuration.of(4, TimeUnit.HOURS))
                .randomBitCount(32)
                .stuckTransactionTimeout(SimpleTimeDuration.of(5, TimeUnit.DAYS))
                .slowLogTriggerMillis(12345)
                .standaloneServer(false)
                .build();

        assertThat(clientMapper.writeValueAsString(lockServerOptions)).isEqualTo(json);
        assertThat(serverMapper.writeValueAsString(lockServerOptions)).isEqualTo(json);

        assertThat(clientMapper.readValue(json, LockServerOptions.class)).isEqualTo(lockServerOptions);
        assertThat(serverMapper.readValue(json, LockServerOptions.class)).isEqualTo(lockServerOptions);
    }
}
```